src/Slider/slider-style.scss

```diff
@@ -6,13 +6,10 @@ $disabled-selected: $gray-500;
 $disabled-unselected: #cccccc;
 $disabled-thumb-bg: #9a9a9a;
 $disabled-thumb-border-color: #8c8c8c;
-
-@mixin transition($props) {
-    -webkit-transition: $props;
-    -moz-transition: $props;
-    -o-transition: $props;
-    transition: $props;
-}
+$slider-width-normal: 24px;
+$slider-height-normal: $slider-width-normal;
+$slider-width-alternative: 56px;
```

Comment:
3.5rem

Reply:
then I should change all the sized in this file to rem. I can't change just this one

Reply:
I don't think we need the slider to increase in size when the user increases the font size. Using pixels is fine in my opinion.